server/odc-service/src/main/java/com/oceanbase/odc/service/permission/database/model/ApplyDatabaseParameter.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.permission.database.model;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import org.springframework.context.i18n.LocaleContextHolder;
+
+import com.oceanbase.odc.common.i18n.I18n;
+import com.oceanbase.odc.core.flow.model.TaskParameters;
+import com.oceanbase.odc.service.connection.database.model.Database;
+import com.oceanbase.odc.service.permission.project.ApplyProjectParameter.ApplyProject;
+
+import lombok.Data;
+
+/**
+ * @author gaoda.xy
+ * @date 2024/1/3 13:54
+ */
+@Data
+public class ApplyDatabaseParameter implements Serializable, TaskParameters {
```

Comment:
we'd better add serialVersionUID, there is already a related issue #1336

Reply:
U are right, done!